gpcontrib/arenadata_toolkit/arenadata_toolkit--1.0--1.1.sql

```diff
@@ -1,3 +1,41 @@
 /* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.0--1.1.sql */
 
 ALTER FUNCTION arenadata_toolkit.adb_get_relfilenodes(tablespace_oid OID) ROWS 30000000;
+
+CREATE OR REPLACE VIEW arenadata_toolkit.adb_skew_coefficients
+AS
+WITH recursive cte AS (
+	WITH all_tables_sizes AS (
+		SELECT t1.autrelname relname, t1.autoid id, inhparent parent_id,
+				arenadata_toolkit.adb_relation_storage_size(t1.autoid) AS size
+			FROM gp_dist_random('gp_toolkit.__gp_user_tables') t1
+			LEFT JOIN pg_inherits t2 ON inhrelid = t1.autoid
+			WHERE autrelstorage != 'x')
```

Comment:
Do we need 'v' and 'f' relation types?

Reply:
I think, we need all types of relation, which can use filesystem.